hr_employee_firstname/i18n/hr_employee_firstname.pot

```diff
@@ -4,8 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 16.0\n"
+"Project-Id-Version: Odoo Server 16.0+e\n"
```

Comment:
Avoid these changes because they will be reverted by the bot anyway.

I'd suggest modifying just the Python code or in fault modifying also the `.pot` file but only terms related to your change.

Reply:
Won't weblate override it anyway and do a msgmerge with the new translations from the .po ? I needed the translation but yes I can remove thoses changes if needed